packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -75,13 +85,15 @@ export class HouseController {
     const userDoc = await this.userStoreService.findOneByFirebaseId(user.uid);
     if (userDoc.house != undefined) {
       const house = await this.houseStoreService.findOne(userDoc.house);
+      const userList = await this.houseStoreService.getUserDto(house.id);
```

Comment:
Just create a new method in `houseStoreService` called `findOneWithUserPopulation` and call this instead on line 87. The method should contain the following: ` return await this.HouseModel.findOne({ _id: id }).populate('users').exec();`. Then there is no need for the `getUserDto` method.